test/powershell/Modules/Microsoft.PowerShell.Management/Set-Location.Tests.ps1

```diff
@@ -55,4 +55,42 @@ Describe "Set-Location" -Tags "CI" {
             (Get-Location).Path | Should Be (Get-PSProvider FileSystem).Home
         }
     }
+
+    Context 'Set-Location with last location history' {
+        
+            It 'Should go to last location when specifying minus as a path' {
+                $initialLocation = Get-Location
+                Set-Location ([System.IO.Path]::GetTempPath())
+                Set-Location -
+                (Get-Location).Path | Should Be ($initialLocation).Path
+            }
+
+            It 'Should go to last location when specifying minus as a path using alias' {
+                $initialLocation = Get-Location
+                Set-Location ([System.IO.Path]::GetTempPath())
+                cd -
```

Comment:
Please remove the test - we shouldn't tests aliases.

Reply:
ok